AFQ/api/group.py

```diff
@@ -916,6 +916,89 @@ def assemble_AFQ_browser(self, output_path=None, metadata=None,
             sublink=page_subtitle_link)
 
 
+class ParallelGroupAFQ(GroupAFQ):
```

Comment:
I don't think we're using inheritance from GroupAFQ:

```suggestion
class ParallelGroupAFQ():
```